packages/blade/src/tokens/theme/theme.d.ts

```diff
@@ -1,7 +1,7 @@
 import { Spacing } from '../global/spacing';
 import { Border } from '../global/border';
 
-export type ColorSchemeNames = 'light' | 'dark';
+export type ColorSchemeNames = 'dark' | 'light';
```

Comment:
Interesting reordering, darkness before light 👀

Reply:
Oh looks like you did this alphabetically, do we have to do this manually?

Reply:
yes I want to turn off this rule. It's annoying 😞